src/main/java/net/fabricmc/loom/util/ModUtils.java

```diff
@@ -36,7 +36,7 @@ public static boolean isMod(File input, ModPlatform platform) {
 		if (platform == ModPlatform.FORGE) {
 			return ZipUtils.contains(input.toPath(), "META-INF/mods.toml");
 		} else if (platform == ModPlatform.QUILT) {
-			return ZipUtils.contains(input.toPath(), "quilt.mod.json");
+			return ZipUtils.contains(input.toPath(), "quilt.mod.json") || ZipUtils.contains(input.toPath(), "fabric.mod.json");
```

Comment:
This shouldn't be required, look at line 54

Reply:
[JarNester.java:110](https://github.com/architectury/architectury-loom/blob/dev/0.11.0/src/main/java/net/fabricmc/loom/build/nesting/JarNester.java#L110) doesn't use shouldRemapJar, it uses isMod directly

Reply:
Then imo JarNester should also check if it's quilt and check if it's fabric as well

Reply:
Done (didn't need to check if it's quilt since that's always true there)